docs/getting_started/create_community.md

```diff
@@ -0,0 +1,53 @@
+# Create and edit communities 
+
+## Types of users 
+
+### Single-tenant users
+For single-tenant users, a workspace is automatically created upon subscribing to Sekoia. Users can only have one Workspace, which does not contain any communities.
+
+### Multi-tenant users
+For multi-tenant users, such as Managed Security Service Providers (MSSPs), a primary workspace is created. These users can add multiple communities to manage their various clients or departments.
+
+!!! info
+    To access multi-tenant mode, contact your Sekoia account executive. Once subscribed, you will be switched to muulti-tenant mode in the following days. 
```

Comment:
```suggestion
    To access multi-tenant mode, contact your Sekoia account executive. Once subscribed, you will be switched to multi-tenant mode in the following days. 
```

Reply:
A sub-community cannot be switched to a multi-tenant mode while remaining a sub-community. Perhaps we should specify this.